packages/react-devtools-shared/src/__tests__/treeContext-test.js

```diff
@@ -872,22 +872,58 @@ describe('TreeListContext', () => {
       );
 
       utils.act(() => TestRenderer.create(<Contexts />));
-      expect(state.selectedElementIndex).toBe(null);
+      let statefulStore = {state, store};
+      expect(statefulStore).toMatchInlineSnapshot(`
```

Comment:
It's a bit unfortunate but when I do `expect({state, store})` I get `Jest: Couldn't locate all inline snapshots.`. 

This is probably the same issue as https://github.com/facebook/jest/issues/6744. Might be worth looking into but sounds like a hard to debug issue.

Reply:
That sucks.

I don't know about this:
```js
let statefulStore = {state, store};
expect(statefulStore).toMatchInlineSnapshot(`
```

This strikes me as pretty weird. Let me think about it 😅

Reply:
I really like the richer snapshots. Maybe it's mostly the naming of this object/serializer that's nagging me.

Reply:
I have an idea that might be janky. Give me a few to try it.

Reply:
Ah! I do have an idea that will work, yes. Give me 10 minutes to push it.

Reply:
Just takin ta while to update more tests to use inline snapshots.